_content/articles/elephant-good-to-think_ohnuma-reiko.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "An elephant good to think: The Buddha in Pārileyyaka forest"
+authors:
+  - "Reiko Ohnuma"
+external_url: "https://www.academia.edu/10975388/An_Elephant_Good_to_Think_The_Buddha_in_P%C4%81rileyyaka_Forest"
```

Comment:
What JIABS site are you looking at? Here's the open access link: https://poj.peeters-leuven.be/secure/POJ/purchaseform.php?id=3078166&sid=

Yes the URL says "purchaseform" but if you go to it you just immediately get redirected to the PDF